google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -43,7 +43,7 @@ appengine.deployment.error.creating.staging.directory=There was an unexpected er
 appengine.generate.config.button=Generate...
 appengine.appyaml.location.label=app.yaml path\:
 appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
-appengine.deployment.error.with.code=Deployment failed with exit code: {0}
+appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK. (See: https://cloud.google.com/sdk/)
```

Comment:
Yes that's definitely right. @meltsufin sorry..what you had before was fine, I'm not sure what I was thinking really.